timemachine/fe/rbfe.py

```diff
@@ -194,9 +196,10 @@ def setup_initial_states(st, host_config, temperature, lambda_schedule, seed):
 
         run_seed = seed + lamb_idx
 
-        if host_config:
+        if host is not None:
```

Comment:
This change is to prevent linter errors like "host system may not be defined" (since it's defined conditionally if `host_config`)